ethereum/datastructures/src/main/java/tech/pegasys/teku/datastructures/state/TransitionCaches.java

```diff
@@ -30,6 +30,7 @@
   private static final int MAX_ACTIVE_VALIDATORS_CACHE = 8;
   private static final int MAX_BEACON_PROPOSER_INDEX_CACHE = 1;
   private static final int MAX_BEACON_COMMITTEE_CACHE = 64 * 64;
+  private static final int MAX_TOTAL_ACTIVE_BALANCE_CACHE = 2;
```

Comment:
Note:  this was originally set to 1 - setting this to 2 so that we can keep a value for the latest block imported + the latest finalized state (which is used by the `WeakSubjectivityCalculator`).